person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] And last but not least, as mentioned on your book class as well, you may need a method to add a rental to a person, so you may need something like `def add_rental(book, date)` and then complete the method with the proper code to allow your person to create a new rental.